packages/kokoas-client/src/pages/projEstimate/estimateDataGrid/renderers/renderUnits.tsx

```diff
@@ -12,16 +12,19 @@ const UnitsSelect = ({
 }: RenderEditCellProps<RowItem>) => {
   const [open, setOpen] = useState(true);
   const { data } = useUnits((u) => {
-    return u.map(({
+    return u?.map(({
       unit,
     }) => ({
       value: unit.value,
     }));
   });
 
+  // Prevent warning from MUI Select #https://github.com/mui/material-ui/issues/18494#issuecomment-782779777
+  if (!data) return null; 
+
   return (
     <Select
-      value={row.unit || undefined}
+      value={row.unit || ''}
```

Comment:
```suggestion
      value={row.unit || undefined}
```

私の方の環境では、修正後はエラーになってしまい、単位のメニューも開きませんでした。
もしPCの動作が異なるようであれば相談させてください。

Reply:
ありがとうございます！

新規で試していて、エラーが出なかったんですが、編集の場合、出ますね。

おかげで、エラーの根本原因を見つかりました。Queryのキーは間違っていました。💦